### PR TITLE
feat(Slider): add isDisabled property

### LINKED ISSUE #5496: feat(Slider): add isDisabled property
Add the ability to disable the Slider using the styles defined in Core. https://github.com/patternfly/patternfly/pull/3879

cc @KKoukiou

### PR DESCRIPTION
**Pull request by KKoukiou**
Closes #5496

### CODE REVIEW

File: packages/react-core/src/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -316,10 +316,11 @@ export const Slider: React.FunctionComponent<SliderProps> = ({
           aria-valuenow={value}
           aria-valuetext={findAriaTextValue()}
           aria-label={thumbAriaLabel}
-          onMouseDown={handleMouseDown}
-          onTouchStart={handleToucheStart}
-          onKeyDown={handleThumbKeys}
-          onClick={onThumbClick}
+          aria-disabled={isDisabled}
```
Comment (mcoker) [inline packages/react-core/src/components/Slider/Slider.tsx:319]: Also need to modify the `tabIndex` above to `-1` (or remove it) when it's disabled so it won't take keyboard focus.

**Review by mcoker — CHANGES_REQUESTED** [review]
Looks great! Just one comment about the `tabindex` on the thumb

**Review by mcarrano — APPROVED** [review]
Looks good to me. Thanks @KKoukiou .

File: packages/react-core/src/components/Slider/Slider.tsx (hunk 1 above)
Comment (KKoukiou) [reply packages/react-core/src/components/Slider/Slider.tsx:319]: Thanks for catching this! Updated with fix.

**Comment by nicolethoen** [comment]
I think this looks great! @jessiehuff I wonder if we don't actually want to change the tabindex, since now the slider is completely undiscoverable to the screen reader? Is there a different way to make sure the screen reader knows its there?

**Comment by mcoker** [comment]
FWIW this models tabindex being removed from other disabled form elements like button, text input, switch, etc - also a [disabled HTML range input/slider](https://codepen.io/mcoker/pen/WNoWbJe). Since it's `aria-disabled`, but not hidden/`aria-hidden`, it should still be discoverable and dimmed by screen readers the way other disabled elements are.

**Review by jessiehuff — COMMENTED** [review]
So I double checked this in VO, JAWS, and NVDA to make sure it worked, and it seems that all of them recognized the disabled slider. VO calls it "dimmed", JAWS and NVDA say "unavailable." There are times that we've managed focus when something interferes with the screen reader seeing it (for example, disabled dropdown items), but otherwise it seems to handle it ok. The only thing I noticed that was confusing in testing was there was no `aria-valuetext` on the disabled example. It was instead announced as 50 which may not be what is intended if a sighted user is seeing 4. I would look into why that isn't getting added with this example.

**Review by mcoker — APPROVED** [review]
LGTM! 🥳

**Review by tlabaj — APPROVED** [review]

**Pull request merged**